fv3core/stencils/fvtp2d.py

```diff
@@ -123,7 +123,7 @@ def __call__(
         corners.copy_corners_x_stencil(
             q, origin=grid.full_origin(), domain=grid.domain_shape_full(add=(0, 0, 1))
         )
-        self.xppm_out(q, crx, self._tmp_fx2, grid.jsd, grid.jed)
+        self.xppm_in(q, crx, self._tmp_fx2, grid.jsd, grid.jed)
```

Comment:
This looks like a dangerous change?

Reply:
This is correct. Somehow it was changed to xppm_out somewhere earlier along the line...